Automattic/jetpack
# Boost: On premium upgrade (if image guide is enabled), request an ISA report

Fixes https://github.com/Automattic/boost-cloud/issues/310

## Proposed changes:

* If Image Guide is enabled after upgrading to premium and landing on the Purchase Successful page, an Image Size Analysis report will be automatically requested.

### Other information:


## Jetpack product discussion



n/a

## Does this pull request change what data or activity we track or use?



no

## Testing instructions:






* Setup this version of Boost (use the free version);
* Make sure you have images on the website;
* Enable Image Guide;
* Upgrade to Premium;
* After you get redirected back to the Purchase Successful page, click continue and scroll down to the Image Size Analysis section
* There should be a report in progress (or maybe completed).

To easily test the above if you have premium, simply open `/wp-admin/admin.php?page=jetpack-boost#purchase-successful`. This will force the Image Size Analysis to request a report.
